lib/utils/bower-version.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
+module.exports = function bowerVersion(packageName) {
+  let bowerJsonPath = path.join(process.cwd(), 'bower.json');
```

Comment:
There's a way to get the project root from the Project object that may be better than `cwd()`. I was thinking of using it to improve the blueprint. I don't remember where I saw it, though.